src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -73,6 +75,16 @@
   private static final String BATCH_SIZE_DISPLAY = "Batch Size";
   private static final int BATCH_SIZE_DEFAULT = 2000;
 
+  public static final String MAX_BULK_SIZE_BYTES_CONFIG = "max.bulk.size.bytes";
+  private static final String MAX_BULK_SIZE_BYTES_DOC =
+      "The maximum size (in bytes) of the bulk request to be processed by the Connector when"
+          + " writing records to Elasticsearch. The default is 10mb, the maximum setting possible."
+          + "If set to -1 or the condition for batch size by number of records is met first, this "
+          + "configuration will be ignored and the setting for batch size by number of records "
+          + "would be used instead.";
```

Comment:
```suggestion
  private static final String BULK_SIZE_BYTES_DOC =
      "The maximum size (in bytes) to be process as a batch when"
          + " writing records to Elasticsearch. Setting to '-1' will disable this configuration. If the condition set by '" + BATCH_SIZE_CONFIG
          + "' is met first, it will be used instead."
```